S1_NRB/processor.py

```diff
@@ -460,15 +468,51 @@ def main(config_file, section_name, debug=False):
                     with bbox(extent, epsg) as dem_box:
                         with Raster(dem_names[i], list_separate=False)[dem_box] as dem_mosaic:
                             dem_mosaic.write(fname_dem, format='GTiff')
-            
-            list_processed = finder(out_dir_scene, ['*'])
+            ###############################################
+            # ETAD correction
+            if config['etad']:
+                print('###### [   ETAD] Scene {s}/{s_total}: {scene}'.format(s=i + 1, s_total=len(ids),
```

Comment:
>do you see a reason why the processor should continue if an error occurs?

What if I want to start a processor run of 20 SLCs over the weekend and only one SLC is potentially causing an error. I think it would be better to let the processor continue and log everything in the log files. In this case the user can easily see in the logs that one scene failed, investigate the reason and try to reprocess that one scene.

>Also, is there a reason why `log.exception` is used for `nrb_processing` instead of `log.error`?

Yes, to also get the error traceback. E.g. this:
```
[2022-03-22 20:28:42,606] [   ERROR] [    NRB] -- ['/geonfs03_vol1/COPA/alps/S1A_IW_SLC__1SDV_20210621T165852_20210621T165919_038441_04893A_D480.zip', '/geonfs03_vol1/COPA/alps/S1A_IW_SLC__1SDV_20210621T165916_20210621T165943_038441_04893A_824A.zip'] -- module 'S1_NRB' has no attribute '__version__'
Traceback (most recent call last):
  File "/home/du23yow/S1_NRB/S1_NRB/processor.py", line 531, in main
    nrb_processing(config=config, scenes=scenes, datadir=config['rtc_dir'], outdir=outdir,
  File "/home/du23yow/S1_NRB/S1_NRB/processor.py", line 286, in nrb_processing
    meta = extract.meta_dict(config=config, target=nrbdir, src_scenes=src_scenes, snap_files=files,
  File "/home/du23yow/S1_NRB/S1_NRB/metadata/extract.py", line 575, in meta_dict
    meta['prod']['processorVersion'] = S1_NRB.__version__
AttributeError: module 'S1_NRB' has no attribute '__version__'
```

Instead of this:
```
[2022-03-22 20:28:42,606] [   ERROR] [    NRB] -- ['/geonfs03_vol1/COPA/alps/S1A_IW_SLC__1SDV_20210621T165852_20210621T165919_038441_04893A_D480.zip', '/geonfs03_vol1/COPA/alps/S1A_IW_SLC__1SDV_20210621T165916_20210621T165943_038441_04893A_824A.zip'] -- 
module 'S1_NRB' has no attribute '__version__'
```

Reply:
>What if I want to start a processor run of 20 SLCs over the weekend and only one SLC is potentially causing an error?

True. But imagine a scenario where `snap.geocode` fails. Then the noise power processing would still be executed and then the NRB processing would be started. It would be better if the processing of this particular scene would just be aborted completely. But as I said, this would be harder to realize right now, so let's not worry about it now.

>Yes, to also get the error traceback.

Right, now I remember that we talked about this difference. In this case I think it would be preferable to always use `log.exception` to have the full traceback available any time some error occurs. Or do you see anything speaking against that?

Reply:
>I don't remember ever having problems with a lack of error traceback for the SNAP parts.

Well the SNAP traceback sure is verbose but not necessarily meaningful :smile:. It is written to a dedicated file anyways. I was more thinking of pyroSAR bugs/errors. All sorts of things can happen, the most frequent one perhaps the unavailability of some DEM download URLs.

I have opened a dedicated internal issue. Let's continue there if necessary.